app/src/main/java/rs/ruffle/PlayerActivity.kt

```diff
@@ -123,9 +123,16 @@ class PlayerActivity : GameActivity() {
                     item.setChecked(true)
                 }
             }
+            val resourceIdExit: Int = items.size + 1
+            menu.add(group, resourceIdExit, Menu.NONE, "Exit")
             popup.setOnMenuItemClickListener { item: MenuItem ->
-                runContextMenuCallback(item.itemId)
-                true
+                if (item.itemId == resourceIdExit) {
+                    finish()
+                    true
```

Comment:
I guess the `true` can be kept out of the branch, as it's the same in both arms.